packages/Webkul/Shop/src/Http/Controllers/Customer/Account/OrderController.php

```diff
@@ -63,7 +63,6 @@ public function printInvoice($id)
     {
         $invoice = $this->invoiceRepository->findOneWhere([
             'id'          => $id,
```

Comment:
Why customer ID removed from here?

Reply:
Refactored invoice retrieval to include order customer inquiry.